AGENTS.md

```
@@ -0,0 +1,187 @@
+
+# Verification
+make verify             # Runs verify-yaml and verify-update
+make verify-yaml        # Validates YAML manifests
+make verify-update      # Ensures generated files are up-to-date
+```
+
+## Architecture
+
+### Core Components
+
+**Main Operator Loop (pkg/cvo/cvo.go)**
```

markdownlint-cli2 (0.18.1)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)